BESMod/Examples/HighOrderModel/BaseClasses.mo

```diff
@@ -0,0 +1,81 @@
+within BESMod.Examples.HighOrderModel;
+package BaseClasses
+  partial model PartialBES_HOM
```

Comment:
Save the package as single files

Reply:
I decided to delete the BaseClasses package, because it contained only one partial model. So I think it is unnecessary.

Reply:
Update: Due to convention, we keep the BaseClasses package and I saved the package as single files.